doc/specifications/drafts/content_edit.md

```diff
@@ -0,0 +1,102 @@
+# Symfony forms content edit
+
+## Synopsis
+repository-forms currently supports creating content without an intermediate draft.
+
+Support must be added for editing content. It implies creating a new version of an existing content,
+rendering a form with the previous version's fields values, and allowing the user to save the draft,
+cancel or publish the new version. It should also allow choosing a language, either for translation
+or 
+
+### Draft management
+Ideally, the draft would only be created when we need it, not on the first display.
+
+However, any interactive action on Fields (custom buttons, etc) will require that the draft
+is created.  Note that the exact same problem exists in createWithoutDraft. We just don't
+have any interactive FieldTypes at the moment.
+
+## Prototypes
+
+### Creating a draft from an existing version
+
+> A route that displays a form allowing to create a draft from an existing content item.
+
+- path: `/content/create/draft/{contentId}/{fromVersionNo}/{fromLanguage}/{toLanguage}`
+- controller action: `ContentEditController::createContentDraftAction`
+
+All the arguments except for `contentId` are optional. Arguments that are specified will
+be used as default values for the form. The draft will only be created when the form is 
+submitted. However, this resource is meant to be used by other parts of the system to create 
+a content draft.
+
+When the form is submitted and a draft is created, a redirection to the content editing route
```

Comment:
Even when publishing directly? A draft should be created in any case, but since the edit draft route is described as showing the edit form: If you publish right after creating a draft (optionally changing some fields), and validation passes, there is no need to show the form again (whether that means skipping the edit draft route, or skipping its form view, is tbd).